grouper/fe/templates/group-join.html

```diff
@@ -21,6 +21,13 @@
                 <h3 class="panel-title">Join Group</h3>
            </div>
             <div class="panel-body">
+                {% if user_is_member %}
+                <div class="alert alert-warning">
+                    You are either already a member of this group or have a pending membership
+                    request. You can also request membership on behalf of a group, but be sure this
+                    is what you intended to do.
```

Comment:
Maybe even be explicit about requesting membership on behalf of a group _usually_ being incorrect?